src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf) format(ttf),
+    url(./fonts/Avenir-Heavy.ttf) format(ttf);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.top__bar {
+  display: flex;
+  flex-direction: column;
+  max-width: 1022px;
+  min-width: 420px;
+  margin: 0 8px;
+}
+
+.search__box {
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties). Fix all cases